docs/migration-guides/1.7-1.8.md

```diff
@@ -0,0 +1,21 @@
+# Migrate from v1.7 to v1.8
+
+>**TIP:** To learn more about upgrading Kyma, read [this](https://kyma-project.io/docs/master/root/kyma/#installation-upgrade-kyma) document.
+
+
+## Monitoring
+
+We have increased the size of Persistent Volume Claims (PVC) for Prometheus to 10GB to prevent it from running out of disk space. Unfortunately, the standard upgrade procedure does not resize the volumes automatically, as the volume expansion through StatefulSets is not supported.
```

Comment:
```suggestion
We have increased the size of Persistent Volume Claims (PVC) for Prometheus to 10GB to prevent it from running out of disk space. The automatic upgrade procedure does not resize the volumes automatically, as the volume expansion through StatefulSets is not supported. As a result, you must trigger the Prometheus PVC size change manually.
```